roles/container-engine/cri-o/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-crio_rhel_repo_base_url: 'https://cbs.centos.org/repos/paas7-openshift-origin311-candidate/x86_64/os/'
+crio_rhel_repo_base_url: 'https://cbs.centos.org/repos/paas7-crio-113-candidate/x86_64/os/'
```

Comment:
1.14 was released. 

https://cbs.centos.org/repos/paas7-crio-114-candidate/x86_64/os/Packages/

Now default k8s is 1.14.1. Above repository should be reasonable.

Reply:
Indeed, CRI-O version should match Kubernetes version installed by kubespray:

| Version - Branch             | Kubernetes branch/version       | Maintenance status |
|------------------------------|---------------------------------|--------------------|
| CRI-O 1.12.x  - release-1.12 | Kubernetes 1.12 branch, v1.12.x | =                  |
| CRI-O 1.13.x  - release-1.13 | Kubernetes 1.13 branch, v1.13.x | =                  |
| CRI-O 1.14.x  - release-1.14 | Kubernetes 1.14 branch, v1.14.x | =                  |
| CRI-O HEAD    - master       | Kubernetes master branch        | ✓                  |

Reply:
IMO, supporting CRI-O 1.14.x is enough. It's experimental feature and currently, updated version seems enough.